infra/cloudtrail/audit_trail.tf

```diff
@@ -0,0 +1,58 @@
+resource aws_s3_bucket dss_audit_logs
+{
+  count = "${var.ENABLE_AUDIT_LOGS == 1 ? 1 : 0}"
+  bucket = "${var.DSS_AUDIT_LOGS_BUCKET}"
+  tags {
+    managedby = "terraform"
+    application = "DSS"
+  }
+  server_side_encryption_configuration {
+    rule {
+      apply_server_side_encryption_by_default {
+        sse_algorithm     = "AES256"
+      }
+    }
+  }
+  lifecycle_rule {
+    id      = "logs"
+    enabled = true
+    transition {
+      days          = 30
+      storage_class = "STANDARD_IA"
+    }
+
+    transition {
+      days          = 90
+      storage_class = "GLACIER"
+    }
+    expiration {
+      days          = 1096  # Three Years
```

Comment:
Why 3 years?

Reply:
That's how long is required by FISMA security standards

Reply:
Sounds good.

Reply:
Created https://github.com/HumanCellAtlas/data-store/issues/1778